README.md

```diff
@@ -1,2 +1,63 @@
 # err-topic-a-day
-An errbot plugin to post a topic a day to a channel for discussion
+
+## Description
+This is a plugin for Errbot (http://errbot.io) to manage a "topic-a-day" channel for daily discussion of topics
+
+## Installation
```

Comment:
We do not need this section. Users can be expected to already have an errbot installation setup.